src/about.njk

```diff
@@ -20,13 +19,17 @@ meta:
             addressLocality: San Francisco
             addressRegion: CA
             postalCode: '94120'
-            addressCountry: USA
+            addressCountry: US
         sameAs:
         - https://www.linkedin.com/company/flowfuse
         - https://www.facebook.com/FlowFuse/
         - https://twitter.com/flowfuseinc
         - https://github.com/FlowFuse
         - https://www.youtube.com/channel/UCbBzP8NZbv3WDtlt4UouA-g
+eleventyComputed:
+    meta:
+        organization:
+            description: "{{ site.messaging.title }}"
```

Comment:
Maybe:
```suggestion
            description: "{{ site.messaging.subtitle }}"
```

Reply:
I think the subtitle is better for the description